app/Http/Kernel.php

```diff
@@ -64,5 +65,6 @@ class Kernel extends HttpKernel
         'throttle' => \Illuminate\Routing\Middleware\ThrottleRequests::class,
         'verified' => \Illuminate\Auth\Middleware\EnsureEmailIsVerified::class,
         'checkAdmin' => \App\Http\Middleware\CheckAdmin::class,
+        'setlocale' => \App\Http\Middleware\SetLocale::class,
```

Comment:
middleware về locale áp dụng cho toàn hệ thống thì nên cho vào global middleware hoặc middlewareGroups phần web em nhé, vì route nào cũng dùng mà

Reply:
dạ em đã sửa ạ

Reply:
để 1 trong 3 cái thôi e